### PR TITLE
Expose card icons through UIImage category

### PR DESCRIPTION
**Pull request by benzguo**
r? @jflinter 

replaces https://github.com/stripe/stripe-ios/pull/288

It doesn't look like the SDK already has a UIImage category?

### CODE REVIEW

File: Stripe/UI/UIImage+Stripe.m [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//  Created by Ben Guo on 1/4/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import "UIImage+Stripe.h"
+#import "STPPaymentCardTextFieldViewModel.h"
+
+@implementation UIImage (Stripe)
+
++ (UIImage *)stp_amexCardImage {
```
Comment (jflinter) [inline Stripe/UI/UIImage+Stripe.m]: I think it'd be better to switch this around - in other words, these methods should just be calling `imageNamed:`, and then STPPaymentCardTextFieldViewModel would depend on this category, not the other way around.

Comment (jflinter) [inline Stripe/UI/UIImage+Stripe.m]: Then you can also move `safeImageNamed` to this category as well.

**Comment by jflinter** [comment]
My mistake; there's no existing UIImage category.

**Comment by jflinter** [comment]
It'd also be nice to have a really quick unit test case for this category that just asserts that the output of all of those methods is non-nil.

File: Stripe/UI/UIImage+Stripe.m (hunk 1 above)
Comment (benzguo) [inline Stripe/UI/UIImage+Stripe.m]: ah, ok – I misread your "all we really want here" and reduced the scope of the changes. This is actually pretty close to what I had in https://github.com/stripe/stripe-ios/pull/288 .

**Comment by benzguo** [comment]
> It'd also be nice to have a really quick unit test case for this category that just asserts that the output of all of those methods is non-nil.

:+1: , added tests

**Comment by jflinter** [comment]
LGTM, sorry I missed this!
